### PR TITLE
added blank field

### PR DESCRIPTION
**Pull request by chelsey-wiley**
Added the create a new card callback to blank the input fields and allow the user to add another card.

### CODE REVIEW

File: client/js/components/CardEditorComponent.js [hunk 1]
```diff
@@ -8,11 +8,14 @@ if (window.FC === undefined) { window.FC = {}; }
       evt.preventDefault();
 
       var cb = () => {
-        ReactRouter.browserHistory.goBack();
+        // ReactRouter.browserHistory.goBack();
```
Comment (tetrakra) [inline client/js/components/CardEditorComponent.js:11]: change this line to

window.alert('you added a card: \r' + this.frontInput.value + " / "+ this.backInput.value);

File: client/js/components/CardEditorComponent.js [hunk 2]
```diff
@@ -28,7 +31,7 @@ if (window.FC === undefined) { window.FC = {}; }
 
         </form>
```
Comment (tetrakra) [inline client/js/components/CardEditorComponent.js:39]: For card submission, add this after form?

  <hr/>
        <button onClick={() => {ReactRouter.browserHistory.goBack();}}>Done</button>

**Review by tetrakra — CHANGES_REQUESTED** [review]
change submitcard class, done button

File: client/js/components/CardEditorComponent.js (hunk 2 above)
Comment (tetrakra) [reply client/js/components/CardEditorComponent.js:39]: <hr/>
        <button onClick={() => {ReactRouter.browserHistory.goBack();}}>Done</button>

File: client/js/components/CardEditorComponent.js (hunk 1 above)
Comment (chelsey-wiley) [reply client/js/components/CardEditorComponent.js:11]: If we add this alert won't it be the same number of clicks as before defeating the purpose of user-ability?

File: client/js/components/CardEditorComponent.js (hunk 2 above)
Comment (chelsey-wiley) [reply client/js/components/CardEditorComponent.js:39]: This makes sense. No endless adding card loop for this app.

**Review by tetrakra — APPROVED** [review]

**Pull request merged**